src/legacy/scss/partials/_shame.scss

```diff
@@ -43,6 +43,9 @@
         z-index: 1001;
         width: 500px;
         height: $lineheight * 17;
+        &--height-50{
```

Comment:
Out of interest, is there a reason for this being `50` specifically? It could be handy to be a bit more descriptive with the name, for example is a larger or taller modal rather than `50`?